package.json

```diff
@@ -144,8 +144,8 @@
     "webpack-dev-server": "1.14.0"
   },
   "peerDependencies": {
-    "babel-polyfill": "6.9.x",
-    "babel-runtime": "6.9.x",
+    "babel-polyfill": "6.x.x",
```

Comment:
I'd be more comfortable allowing something like `>=6.9.0 <7.0` since we know for sure 6.9.x works and we want to allow new versions.

Makes sense?